orchagent/portsorch.cpp

```diff
@@ -1155,6 +1261,9 @@ bool PortsOrch::addBridgePort(Port &port)
 {
     SWSS_LOG_ENTER();
 
+    if (port.m_bridge_port_id)
+        return true;
```

Comment:
use {} even for one line statement.

Reply:
I think we should print some warning here, since we should not add a port to a bridge if the port is in a bridge already.

---
In reply to: [142000240](https://github.com/Azure/sonic-swss/pull/327#discussion_r142000240) [](ancestors = 142000240)

Reply:
done